### ISSUE TITLE
Carousel images need alternative text attribute

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13430]
**Describe the bug**

- Automated tests reveal missing alterative text in the carousel's images.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**

1. Go to Homepage
2. Inspect Carousel images
3. Notice missing `alt` attribute

**Expected behavior**

- The link describes the image already for the purpose of the carousel, so in this case, the `alt` attribute can be empty, but it should be present.

**Screenshots**
![Screen Shot 2021-08-05 at 12 07 04 PM](https://user-images.githubusercontent.com/15113219/128561930-e8cddddb-581b-4dbd-9683-b73afd96ddb2.png)

**Comment by saad-mo** [comment]
EDIT: Solutions is to add `[alt]="product.name"` to `cx-media` in `product-carousel.component.html`

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

<img width="462" alt="Screen Shot 2022-01-17 at 2 26 38 PM" src="https://user-images.githubusercontent.com/45409515/149829337-e3b6d9b9-96c3-49e3-a9dd-7d5988160c74.png">
